### PR TITLE
Waiv 2308,2305,2295

### PR DESCRIPTION
**Pull request by oksanana**
Fixes #

### Changes

Summary of changes you made.

* Change 1
* Change 2

### Test plan

Explain how to test changes you made.

* [ ] Step 1
* [ ] Step 2

### Demo (optional)

Sreenshots (before & after) or video that show result of your work.

### CODE REVIEW

**Comment by oksanana** [comment]
2308, 2305, 2295, 2299

File: src/client/rewards/PaymentCard/PaymentCard.js [hunk 1]
```diff
@@ -34,24 +34,44 @@ const PaymentCard = ({ intl, payable, name, alias, history, path, match }) => {
...
+        })}
+        {` ${payable && payable.toFixed(2)} HIVE`}
+      </Action>
+    );
+  };
 
-  if (match.path === '/rewards/receivables') {
-    renderTransferButton = <span>{` ${payable && payable.toFixed(2)} HIVE`}</span>;
-  }
+  const renderTransferButton = useMemo(() => getTransferButton(), [match.path]);
```
Comment (milklim) [inline src/client/rewards/PaymentCard/PaymentCard.js:74]: you can create `TransferButton` component

File: src/client/object/Catalog/CatalogWrap.js [hunk 2]
```diff
@@ -113,6 +113,19 @@ class CatalogWrap extends React.Component {
...
+        'tagCategory',
+        'categoryItem',
+        'galleryItem',
+      ]).then(data => {
+        myWobject = data;
+        const requiredObject = this.getRequiredObject(myWobject, match);
+        if (requiredObject) {
+          this.getPropositions({ userName, match, requiredObject, sort });
+        }
+      });
```
Comment (milklim) [inline src/client/object/Catalog/CatalogWrap.js:128]: what task is it?

Comment (milklim) [reply src/client/object/Catalog/CatalogWrap.js:128]: 2299

File: src/client/object/Catalog/CatalogWrap.js [hunk 3]
```diff
@@ -113,6 +113,19 @@ class CatalogWrap extends React.Component {
       if (requiredObject) {
         this.getPropositions({ userName, match, requiredObject, sort });
       }
+    } else {
+      let myWobject;
```
Comment (milklim) [inline src/client/object/Catalog/CatalogWrap.js:117]: remove this var

File: src/client/object/Catalog/CatalogWrap.js [hunk 4]
```diff
@@ -113,6 +113,19 @@ class CatalogWrap extends React.Component {
...
+      getObject(match.params.name, this.props.userName, [
+        'tagCategory',
+        'categoryItem',
+        'galleryItem',
+      ]).then(data => {
```
Comment (milklim) [inline src/client/object/Catalog/CatalogWrap.js:122]: rename data -> wObject

**Pull request merged**